themes/custom_theme/valencia-theme/src/css/_custom.scss

```diff
@@ -119,12 +104,16 @@ ul {
 }
 
 .page-header {
-	height: 90vh;
+	height: auto;
+	min-height: 100vh;
 	margin: 0;
 	padding: 0;
 	border: 0;
 	background-position: center center;
 	background-size: cover;
+	@media #{$screen-md} {
```

Comment:
We should put all media queries grouped together at the bottom of the file to keep everything together.